packages/react/README.md

```diff
@@ -65,3 +77,93 @@ function Page() {
 
 export default App;
 ```
+
+### Multiple Providers and Scoping
+
+Multiple providers and scoped clients can be configured by passing a `clientName` to the `OpenFeatureProvider`:
+
+```tsx
+// Flags within this scope will use the a client/provider associated with `myClient`,
+function App() {
+  return (
+    <OpenFeatureProvider clientName={'myClient'}>
+      <Page></Page>
+    </OpenFeatureProvider>
+  );
+}
+```
+
+This is analogous to:
+
+```ts
+OpenFeature.getClient('myClient');
+```
+
+### Re-rendering with Context Changes
+
+By default, if the OpenFeature [evaluation context](https://openfeature.dev/docs/reference/concepts/evaluation-context) is modified, components will be re-rendered.
```

Comment:
Minor - should we also highlight that react sdk implements static context paradigm [1] of the spec? 

[1] - https://openfeature.dev/specification/glossary#static-context-paradigm

Reply:
Good call, yes!

Reply:
Done